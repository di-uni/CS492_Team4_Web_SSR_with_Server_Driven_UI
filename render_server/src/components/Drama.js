import React from 'react'
import Link from '@mui/material/Link';
import MoreVert from './MoreVert'
import styles from './drama.module.css'

import { Card, CardContent, CardHeader, Divider, Box, Paper } from '@mui/material';

export default function Drama({props}) {
    const [anchorEl, setAnchorEl] = React.useState(null);
    const open = Boolean(anchorEl);
    const handleClick = (event) => {
        setAnchorEl(event.currentTarget);
        console.log(event.currentTarget)
    };
    const handleClose = () => {
        setAnchorEl(null);
    };

    var link;
    if (typeof window !== "undefined") {
        link = window.location.href;
        console.log(link);
    }

    const parentRef   = React.useRef(null);
    const childrenRef = React.useRef(null);
    var parentWidth;

    React.useEffect ( () => {
        if(parentRef.current){
            let parentHeight = parentRef.current.offsetHeight;
            parentWidth  = parentRef.current.offsetWidth;
            console.log("parent width:", parentWidth);
        }
        if(childrenRef.current){
            let childrenHeight = childrenRef.current.offsetHeight;
            let childrenWidth  = childrenRef.current.offsetWidth;
            console.log("children width:", childrenWidth);
        }
    }, [parentRef, childrenRef]);
    
    return (
        <>
        <div className={ styles.page }>
        <Card sx={{ maxWidth: 768 }} style={{backgroundColor: "#F1EDEB" }} variant='outlined' square  ref={ parentRef }>
        {/* <div className={styles.wrapHeader}>
        <div className={styles.basicInfoHeader}>
            <div>
                <Link underline="none">
                // <Link href={dramaURL} underline="none">
                    <div className={styles.dramaName}>술꾼도시여자들</div>
                </Link>
                <Link underline="none">
                // </Link><Link href={categoryURL} underline="none"> 
                    <div className={styles.dramaCategory}>웹드라마</div>
                </Link>
                <div className={styles.menu}>
                    전체
                    <div className={styles.verticalDivider}></div>
                    기본정보
                    <div className={styles.verticalDivider}></div>
                    등장인물
                    <div className={styles.verticalDivider}></div>
                    공식영상
                    <div className={styles.verticalDivider}></div>
                    함께 볼만한 웹드라마
                </div>
            </div>
            <Box className={styles.box} sx={{m: 0.5}}/>
            <MoreVert props = {{
                "open": open, 
                "anchorEl": anchorEl, 
                "handleClick": handleClick, 
                "handleClose": handleClose,
                "url": link,
                "title": "술꾼도시여자들 공식영상",
                "activeColor": "rgba(145,120,103,1)",
                "inactiveColor": "rgba(145,120,103,0.5)"
            }}/>
        </div>
        </div> */}
        {/* <div ref={ childrenRef }> */}
        <Card className={styles.basicInfoHeader} sx={{ maxWidth: 768, maxHeight: 122 }} style={{backgroundColor: "#F1EDEB" }} variant='outlined' square  ref={ parentRef }>
        <CardHeader 
            sx={{ paddingRight: '8px', 
                '& .MuiCardHeader-action': { padding: 0}
            }}
            // style={{backgroundColor: "#F1EDEB" }}
            title= {
                // <div className={styles.headerInfo}>
                <div>
                    <Link underline="none">
                    {/* // <Link href={dramaURL} underline="none"> */}
                        <div className={styles.dramaName}>술꾼도시여자들</div>
                    </Link>
                    <Link underline="none">
                    {/* // <Link href={categoryURL} underline="none"> */}
                        <div className={styles.dramaCategory}>웹드라마</div>
                    </Link>
                    <div className={styles.menu}>
                        전체
                        <div className={styles.verticalDivider}></div>
                        기본정보
                        <div className={styles.verticalDivider}></div>
                        등장인물
                        <div className={styles.verticalDivider}></div>
                        공식영상
                        <div className={styles.verticalDivider}></div>
                        함께 볼만한 웹드라마
                    </div>
                    <Box sx={{m: 0.5}}/>
                </div>
            }
            action = {
                <MoreVert props = {{
                    "open": open, 
                    "anchorEl": anchorEl, 
                    "handleClick": handleClick, 
                    "handleClose": handleClose,
                    "url": link,
                    "title": "술꾼도시여자들 공식영상",
                    "activeColor": "rgba(145,120,103,1)",
                    "inactiveColor": "rgba(145,120,103,0.5)"
                }}
                />
            }
            style={{ textAlign: 'left'}}
        />
        </Card>
        <Paper className={styles.innerContent}
            sx={{ mt: '122px', maxWidth:746, height: 1000, borderRadius: '8px' }}>
            <Paper 
            sx={{ maxWidth:716, height: 464, borderRadius: '6px'}}
            style= {{ background: "#D2C9C4" }}
            >
        {/* // {drama_videos&&drama_videos.map((video) => <PostCard key={video.title} props={video} view={{"viewType": ViewType.NEWS }}/>)} */}
            </Paper>
        </Paper>
      </Card>
      </div>

      {/* <MoreContent props={{'view_option': "뉴스",'more_link':news_more}}/> */}
        </>
    )
}