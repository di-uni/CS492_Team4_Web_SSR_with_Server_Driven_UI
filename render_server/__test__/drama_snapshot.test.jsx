import React from 'react'
import renderer from 'react-test-renderer'
import Drama from '../src/components/Drama'

it('renders homepage unchanged', () => {
    const drama = {
      "header": {
          "title": "술꾼도시여자들",
          "href": "https://m.tving.com/app/theme.tving?pocCode=POCD0600&themeSeq=109682&utm_source=portal&utm_medium=search&utm_campaign=WorkLaterDrinkNow&utm_content=eventpage",
          "category": "웹드라마",
          "tablist": ["전체", "기본정보", "등장인물", "공식영상", "함께 볼만한 웹드라마"],
          "categoryURL": "https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blFB&x_csa=%7B%22pkid%22%3A%22356%22%7D&qvt=0&query=%EC%9B%B9%EB%93%9C%EB%9D%BC%EB%A7%88",
          "tablisthref": ["https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blZH&x_csa=%7B%22pkid%22%3A648%7D&pkid=648&os=19371944&x_nqx=%7B%22theme%22%3A%22culturalassets%22%2C%22pkid%22%3A%22648%22%2C%22os%22%3A%2219371944%22%7D&qvt=0&query=%EA%B2%BD%EC%A3%BC%20%EB%B6%88%EA%B5%AD%EC%82%AC", "https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blZH&x_csa=%7B%22pkid%22%3A648%7D&pkid=648&os=19371944&x_nqx=%7B%22theme%22%3A%22culturalassets%22%2C%22pkid%22%3A%22648%22%2C%22os%22%3A%2219371944%22%7D&qvt=0&query=%EA%B2%BD%EC%A3%BC%20%EB%B6%88%EA%B5%AD%EC%82%AC%20%EA%B8%B0%EB%B3%B8%EC%A0%95%EB%B3%B4", "https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blZH&x_csa=%7B%22pkid%22%3A648%7D&pkid=648&os=19371944&x_nqx=%7B%22theme%22%3A%22culturalassets%22%2C%22pkid%22%3A%22648%22%2C%22os%22%3A%2219371944%22%7D&qvt=0&query=%EA%B2%BD%EC%A3%BC%20%EB%B6%88%EA%B5%AD%EC%82%AC%20%ED%8F%AC%ED%86%A0", "https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blZH&x_csa=%7B%22pkid%22%3A648%7D&pkid=648&os=19371944&x_nqx=%7B%22theme%22%3A%22culturalassets%22%2C%22pkid%22%3A%22648%22%2C%22os%22%3A%2219371944%22%7D&qvt=0&query=%EA%B2%BD%EC%A3%BC%20%EB%B6%88%EA%B5%AD%EC%82%AC%20%EC%A3%BC%EB%B3%80%20%EB%AC%B8%ED%99%94%EC%9E%AC", "https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blZH&x_csa=%7B%22pkid%22%3A648%7D&pkid=648&os=19371944&x_nqx=%7B%22theme%22%3A%22culturalassets%22%2C%22pkid%22%3A%22648%22%2C%22os%22%3A%2219371944%22%7D&qvt=0&query=%EA%B2%BD%EC%A3%BC%20%EB%B6%88%EA%B5%AD%EC%82%AC%20%ED%95%A8%EA%BB%98%20%EC%B0%BE%EC%9D%80%20%EB%AC%B8%ED%99%94%EC%9E%AC"]
      },
      "videos": {
          "showMore": "https://tv.naver.com/v/22495407/list/739104",
          "videoData": [
              {
                  "title": "[술꾼도시여자들] 🍺술도녀 보내기 아쉬운 분들을 위한 역대급 분량 하드털이 메이킹",
                  "publisher": "네이버TV",
                  "viewCount": "4,534",
                  "date": "2021.11.30.",
                  "videoURL": "https://tv.naver.com/v/23826103",
                  "videoLength": "24:36",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211130_183%2FagfL7_1638280758457AMaev_JPEG%2F1638280716455.jpg"
              },
              {
                  "title": "[술꾼도시여자들] 이선빈X한선화에게 사랑받는(?) 정은지💓 지구방위대씬 비하인드",
                  "publisher": "네이버TV",
                  "viewCount": "2,066",
                  "date": "2021.11.30.",
                  "videoURL": "https://tv.naver.com/v/23826072",
                  "videoLength": "12:59",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211130_262%2F3Hciy_1638280544584Oo8ki_JPEG%2F1638280476322.jpg"
              },
              {
                  "title": "[술꾼도시여자들] 이선빈X최시원 베드씬부터 한선화의 데이트, 정은지X동생의 꽁냥 모먼트",
                  "publisher": "네이버TV",
                  "viewCount": "16,894",
                  "date": "2021.11.22.",
                  "videoURL": "https://tv.naver.com/v/23677808",
                  "videoLength": "14:10",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211122_75%2Fk3TeH_1637590335132ofURX_JPEG%2F1637590244966.jpg"
              },
              {
                  "title": "[술꾼도시여자들] 술도녀에게 닥친 인생 최대 시련ㅣ11화 예고",
                  "publisher": "네이버TV",
                  "viewCount": "22,318",
                  "date": "2021.11.22.",
                  "videoURL": "https://tv.naver.com/v/23677888",
                  "videoLength": "00:56",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211122_19%2FAMixR_1637590566041ADSeJ_JPEG%2F1637590533723.jpg"
              },
              {
                  "title": "[술꾼도시여자들] 정은지, 이선빈 얼굴에 냉수 뿌린 이유는? 열연 돋보인 장례식씬 메이킹",
                  "publisher": "네이버TV",
                  "viewCount": "4,220",
                  "date": "2021.11.22.",
                  "videoURL": "https://tv.naver.com/v/23677736",
                  "videoLength": "10:15",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211122_296%2FH9CpO_1637590120907Iy20U_JPEG%2F1637590004889.jpg"
              },
              {
                  "title": "[술꾼도시여자들] 수험생분들을 위한 술도녀팀의 응원💓 수능대박나고 내년에 술 먹자!",
                  "publisher": "네이버TV",
                  "viewCount": "18,626",
                  "date": "2021.11.15.",
                  "videoURL": "https://tv.naver.com/v/23538553",
                  "videoLength": "00:56",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211115_49%2FgZblp_1636968668450d8h5q_JPEG%2F1636968588451.jpg"
              },
              {
                  "title": "[술꾼도시여자들]갑작스러운 이선빈 아빠의 죽음, 곁을 지키는 한선화와 정은지ㅣ9화 예고",
                  "publisher": "네이버TV",
                  "viewCount": "72,356",
                  "date": "2021.11.15.",
                  "videoURL": "https://tv.naver.com/v/23538482",
                  "videoLength": "00:33",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211115_225%2F2J4T2_1636968444441Q9NJz_JPEG%2F1636968387611.jpg"
              },
              {
                  "title": "[술꾼도시여자들] '가끔 넘어져도 괜찮아!' 정은지의 청춘을 위한 인생 한잔ㅣ스페셜MV",
                  "publisher": "네이버TV",
                  "viewCount": "18,430",
                  "date": "2021.11.15.",
                  "videoURL": "https://tv.naver.com/v/23538046",
                  "videoLength": "03:16",
                  "thumbImg": "https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211115_176%2F5qFdc_16369672438573DSkl_JPEG%2F1636967136238.jpg"
              }
          ]
      }
  }
  const tree = renderer.create(<Drama props={drama}/>).toJSON()
  expect(tree).toMatchSnapshot()
})