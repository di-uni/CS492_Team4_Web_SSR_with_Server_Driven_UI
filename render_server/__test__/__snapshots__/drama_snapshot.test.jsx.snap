// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
Array [
  <div
    className="MuiPaper-root MuiPaper-none MuiCard-root stickyHeader css-1yeasr5-MuiPaper-root-MuiCard-root"
    style={
      Object {
        "backgroundColor": "#F1EDEB",
      }
    }
  >
    <div
      className="MuiCardHeader-root css-18x4way-MuiCardHeader-root"
      style={
        Object {
          "textAlign": "left",
        }
      }
    >
      <div
        className="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
      >
        <span
          className="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
        >
          <div>
            <div
              className="dramaName"
            >
              <a
                className="dramaName"
                href="https://m.tving.com/app/theme.tving?pocCode=POCD0600&themeSeq=109682&utm_source=portal&utm_medium=search&utm_campaign=WorkLaterDrinkNow&utm_content=eventpage"
                underline="none"
              >
                술꾼도시여자들
              </a>
            </div>
            <a
              className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineNone css-1oaaite-MuiTypography-root-MuiLink-root"
              href="https://m.search.naver.com/search.naver?where=m&sm=mtb_etc&mra=blFB&x_csa=%7B%22pkid%22%3A%22356%22%7D&qvt=0&query=%EC%9B%B9%EB%93%9C%EB%9D%BC%EB%A7%88"
              onBlur={[Function]}
              onFocus={[Function]}
            >
              <div
                className="dramaCategory"
              >
                웹드라마
              </div>
            </a>
            <div
              className="btnmenu"
            >
              <div
                className="btnmenu"
              >
                <button
                  className="firstunselectedMenu"
                  onClick={[Function]}
                >
                  전체
                </button>
                <div
                  className="verticalDivider"
                />
              </div>
              <div
                className="btnmenu"
              >
                <button
                  className="unselectedMenu"
                  onClick={[Function]}
                >
                  기본정보
                </button>
                <div
                  className="verticalDivider"
                />
              </div>
              <div
                className="btnmenu"
              >
                <button
                  className="unselectedMenu"
                  onClick={[Function]}
                >
                  등장인물
                </button>
              </div>
              <div
                className="btnmenu"
              >
                <button
                  className="selectedMenu"
                  onClick={[Function]}
                >
                  공식영상
                </button>
              </div>
              <div
                className="btnmenu"
              >
                <button
                  className="unselectedMenu"
                  onClick={[Function]}
                >
                  함께 볼만한 웹드라마
                </button>
              </div>
            </div>
            <div
              className="MuiBox-root css-1kxrbkv"
            />
          </div>
        </span>
      </div>
      <div
        className="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
      >
        <div>
          <button
            className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-bnzas2-MuiButtonBase-root-MuiIconButton-root"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "backgroundColor": "transparent",
              }
            }
            tabIndex={0}
            type="button"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="MoreVertIcon button"
              focusable="false"
              style={
                Object {
                  "color": "rgba(145,120,103,0.5)",
                }
              }
              viewBox="0 0 24 24"
            >
              <path
                d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="MuiPaper-root MuiPaper-outlined MuiCard-root contents css-b4hzzr-MuiPaper-root-MuiCard-root"
    style={
      Object {
        "backgroundColor": "#F1EDEB",
      }
    }
  >
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1umbu0d-MuiPaper-root"
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1jca0ss-MuiPaper-root"
        style={
          Object {
            "background": "#FFFFFF",
          }
        }
      >
        <a
          href="https://tv.naver.com/v/23826103"
        >
          <div
            className="thumb"
          >
            <img
              alt="video"
              className="img"
              src="https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211130_183%2FagfL7_1638280758457AMaev_JPEG%2F1638280716455.jpg"
            />
            <div
              className="icon"
            >
              <img
                alt="icon"
                color="transparent"
                height="32px"
                src="/images/Bulguksa/video.png"
                width="28px"
              />
            </div>
            <span
              className="time"
            >
              24:36
            </span>
          </div>
        </a>
        <div
          className="videoInfo"
        >
          <a
            href="https://tv.naver.com/v/23826103"
          >
            <div
              className="videoTitle"
            >
              [술꾼도시여자들] 🍺술도녀 보내기 아쉬운 분들을 위한 역대급 분량 하드털이 메이킹
            </div>
            <div
              className="subInfo"
            >
              네이버TV
              <div
                className="verticalDivider2"
              />
              4,534
              <div
                className="verticalDivider2"
              />
              2021.11.30.
            </div>
          </a>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1jca0ss-MuiPaper-root"
        style={
          Object {
            "background": "#FFFFFF",
          }
        }
      >
        <a
          href="https://tv.naver.com/v/23826072"
        >
          <div
            className="thumb"
          >
            <img
              alt="video"
              className="img"
              src="https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211130_262%2F3Hciy_1638280544584Oo8ki_JPEG%2F1638280476322.jpg"
            />
            <div
              className="icon"
            >
              <img
                alt="icon"
                color="transparent"
                height="32px"
                src="/images/Bulguksa/video.png"
                width="28px"
              />
            </div>
            <span
              className="time"
            >
              12:59
            </span>
          </div>
        </a>
        <div
          className="videoInfo"
        >
          <a
            href="https://tv.naver.com/v/23826072"
          >
            <div
              className="videoTitle"
            >
              [술꾼도시여자들] 이선빈X한선화에게 사랑받는(?) 정은지💓 지구방위대씬 비하인드
            </div>
            <div
              className="subInfo"
            >
              네이버TV
              <div
                className="verticalDivider2"
              />
              2,066
              <div
                className="verticalDivider2"
              />
              2021.11.30.
            </div>
          </a>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1jca0ss-MuiPaper-root"
        style={
          Object {
            "background": "#FFFFFF",
          }
        }
      >
        <a
          href="https://tv.naver.com/v/23677808"
        >
          <div
            className="thumb"
          >
            <img
              alt="video"
              className="img"
              src="https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211122_75%2Fk3TeH_1637590335132ofURX_JPEG%2F1637590244966.jpg"
            />
            <div
              className="icon"
            >
              <img
                alt="icon"
                color="transparent"
                height="32px"
                src="/images/Bulguksa/video.png"
                width="28px"
              />
            </div>
            <span
              className="time"
            >
              14:10
            </span>
          </div>
        </a>
        <div
          className="videoInfo"
        >
          <a
            href="https://tv.naver.com/v/23677808"
          >
            <div
              className="videoTitle"
            >
              [술꾼도시여자들] 이선빈X최시원 베드씬부터 한선화의 데이트, 정은지X동생의 꽁냥 모먼트
            </div>
            <div
              className="subInfo"
            >
              네이버TV
              <div
                className="verticalDivider2"
              />
              16,894
              <div
                className="verticalDivider2"
              />
              2021.11.22.
            </div>
          </a>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1jca0ss-MuiPaper-root"
        style={
          Object {
            "background": "#FFFFFF",
          }
        }
      >
        <a
          href="https://tv.naver.com/v/23677888"
        >
          <div
            className="thumb"
          >
            <img
              alt="video"
              className="img"
              src="https://search.pstatic.net/common?type=f&size=327x180&quality=75&direct=true&src=https%3A%2F%2Fphinf.pstatic.net%2Ftvcast%2F20211122_19%2FAMixR_1637590566041ADSeJ_JPEG%2F1637590533723.jpg"
            />
            <div
              className="icon"
            >
              <img
                alt="icon"
                color="transparent"
                height="32px"
                src="/images/Bulguksa/video.png"
                width="28px"
              />
            </div>
            <span
              className="time"
            >
              00:56
            </span>
          </div>
        </a>
        <div
          className="videoInfo"
        >
          <a
            href="https://tv.naver.com/v/23677888"
          >
            <div
              className="videoTitle"
            >
              [술꾼도시여자들] 술도녀에게 닥친 인생 최대 시련ㅣ11화 예고
            </div>
            <div
              className="subInfo"
            >
              네이버TV
              <div
                className="verticalDivider2"
              />
              22,318
              <div
                className="verticalDivider2"
              />
              2021.11.22.
            </div>
          </a>
        </div>
      </div>
    </div>
    <hr
      className="MuiDivider-root MuiDivider-fullWidth css-1buw4dk-MuiDivider-root"
    />
    <a
      onClick={[Function]}
      style={
        Object {
          "cursor": "pointer",
        }
      }
    >
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-btjtr2-MuiPaper-root"
        style={
          Object {
            "backgroundColor": "#FAFBFB",
          }
        }
      >
        <div
          className="viewAll"
        >
          펼쳐보기
          <test-file-stub
            className="iconDown"
            style={
              Object {
                "height": "16px",
                "stroke": "#8A8D8F",
                "width": "9px",
              }
            }
          />
        </div>
      </div>
    </a>
  </div>,
]
`;
